Mod0/computer-setup.md

```diff
@@ -0,0 +1,138 @@
+- [Install Zoom](#install-zoom)
+- [Snap your Windows](#snap-your-windows)
+
+<br>
+
+### Install Chrome
+
+Chrome includes a set of Developer Tools that will come in handy down the road. Additionally, it is always on the cutting edge of supporting new web technologies.
+
+**Action Steps:**
+- _If you haven't already,_ [download and install Chrome](https://www.google.com/chrome/).
+- Set Chrome as your default browser: 
+  - In your Chrome window, select the More menu by clicking on the three dots.
+  - Select `Settings` from the list.
+  - In the Default browser section on the left, click `Make default`.
+
+<br>
+
+### Install Slack
+
+Slack is our main communication tool at Turing. **You will receive an invitation to join the Turing Workspace 2 weeks before your Mod 0 start date**. If you are not familiar with it, it may seem overwhelming at first. There are a lot of tips and tricks you'll pick up along the way.
+
+**Action Steps:**
+- [Download and install the Slack desktop application](https://slack.com/downloads/windows)
+
+To ensure that you are prepared for your first session:
+- Watch the [Slack Tutorial Video](https://www.youtube.com/watch?v=tfq333EpWgM).
+- Upload a clear and appropriate headshot for your profile image.
+- [Set up your Slack profile](https://www.loom.com/share/6f10b796c90f40c09594899d71bf401a) to include your cohort, program, pronouns, and time zone.
+- It's a good idea to get in the habit of checking Slack daily starting now.
+
+<br>
+
+### Create a GitHub Account
+
+GitHub is a widely used platform that hosts code projects and allows for collaborative workflows. You will spend a lot of time on this site and become fluent in using it as a tool.
+
+**Action Steps:**
+- [Create an account on GitHub](https://github.com/). Please use your Turing email address.
+- [Update your GitHub Profile](https://github.com/settings/profile) and be sure that you choose a username, status, profile info, and picture that are appropriate for a professional setting. GitHub is a platform where future/potential employers will be reviewing your work.
+- [Bookmark this webpage on Chrome](https://support.google.com/chrome/answer/188842?hl=en&co=GENIE.Platform%3DDesktop), so that you can easily find it again later.
+
+<br>
+
+### Create a Replit Account
+
+[Replit](https://replit.com/~) provides an online platform that allows us to write code and see the results almost immediately - all inside of one browser tab! There are many other similar (and awesome) tools available but replit is used regularly in Mods 1 and 2, so Mod 0 provides an introduction and some exposure.
+
+Follow the steps below (if they are not necessary because you have already done them or have experience with replit, you can skip this):
+- Create a free account using your Turing email address.
+- [Bookmark this webpage on Chrome](https://support.google.com/chrome/answer/188842?hl=en&co=GENIE.Platform%3DDesktop), so that you can easily find it again later.
```

Comment:
I think this is a duplicate of line 57 above.

Reply:
Looks like it is also duplicated on line 77 below.

Reply:
This was a copy/paste from the existing Mod 0 computer setup. It's a link to the "how to bookmark on Chrome" tutorial.